starter_code/elevator.js

```diff
@@ -3,17 +3,68 @@ class Elevator {
     this.floor      = 0;
     this.MAXFLOOR   = 10;
     this.requests   = [];
+    this.direction = "";
```

Comment:
it's better to set the direction to be "up" so you don't need to change it afterwards in the floorUp